03-rnaseq/differential-expression_rnaseq_01.Rmd

```diff
@@ -0,0 +1,417 @@
+---
+title: "Differential Expression - RNA-seq"
+author: "CCDL for ALSF"
+date: "`r format(Sys.time(), '%B %Y')`"
+output:   
+  html_notebook:
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+# Purpose of the analysis:
+
+<!--WHAT DOES THIS ANALYSIS DO AND WHEN WOULD YOU USE IT?-->
```

Comment:
We introduce these to help reviewing because it's otherwise hard to see what information is new and what is from the template. I don't like it but I don't have a good alternative. I guess we can keep those in as we are reviewing and then delete before merging?

Reply:
I think people are likely to forget to do that last step but maybe I'm wrong. I usually look at the template at the same time when I review.